bash-preexec.sh

```diff
@@ -281,7 +281,7 @@ __bp_install_after_session_init() {
     # trap. __bp_install sets PROMPT_COMMAND to its final value, so these are only
     # invoked once.
     # It's necessary to clear any existing DEBUG trap in order to set it from the install function.
-    PROMPT_COMMAND='__bp_trap_string=$(trap -p DEBUG); trap DEBUG; __bp_install'
+    PROMPT_COMMAND=$'\n__bp_trap_string=$(trap -p DEBUG); trap DEBUG; __bp_install\n'
```

Comment:
Overall, lgtm; I verified that $' ' quoting should work in the expected way. I think it would be better to replace all the ';'s with newlines, however.

Reply:
Also - it might make sense to double-quote the process substitution:

...__bp_trap_string="$(trap -p DEBUG)"...

Reply:
Adding the quotes. I feel like the semicolons are appropriate considering it's one line of logic here.

Reply:
Reconsidering :/  I'll change to all newlines. That's an easy convention to adhere to.